### PR TITLE
Update cert.rst

### PR DESCRIPTION
**Pull request by kucerakk**
comment

### CODE REVIEW

File: api-ref/source/classic/cert.rst [hunk 1]
```diff
@@ -8,7 +8,8 @@ Creating a Certificate
 Function
 ^^^^^^^^
 
-This API is used to create a certificate for an HTTPS listener.
+This API is used to create a certificate for an HTTPS listener. **THIS IS MY MODIFICATION**
+This should not be here...
```
Comment (vladimirhasko) [inline api-ref/source/classic/cert.rst]: This is wrong, please remove

Comment (vladimirhasko) [reply api-ref/source/classic/cert.rst]: Probably you made a typo

File: api-ref/source/classic/cert.rst [hunk 2]
```diff
@@ -8,7 +8,8 @@ Creating a Certificate
 Function
 ^^^^^^^^
 
-This API is used to create a certificate for an HTTPS listener.
+This API is used to create a certificate for an HTTPS listener. **THIS IS MY MODIFICATION**
```
Comment (vladimirhasko) [inline api-ref/source/classic/cert.rst:11]: I like this

**Review by vladimirhasko — CHANGES_REQUESTED** [review]
There is some wrong content, please remove it

File: api-ref/source/classic/cert.rst (hunk 1 above)
Comment (kucerakk) [reply api-ref/source/classic/cert.rst]: corrected

**Pull request closed without merging**